src/attributes.jl

```diff
@@ -366,6 +387,13 @@ Possible values are:
 
 ## Constraint attributes
 
+"""
+    ListOfConstraintAttributesSet()
+
+A `Vector{AbstractConstraintAttribute}` of all constraint attributes that were set to the instance.
```

Comment:
This may need to be redefined to include `F` and `S` as parameters depending on the outcome of #191.